.github/workflows/fault.yml

```diff
@@ -0,0 +1,111 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: fault
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/**'
+  pull_request:
+    branches:
+      - main
+      - 'release/**'
+  workflow_dispatch:
+env:
+    ContainerRegistry: "ghcr.io"
+    ContainerRegistryRepo: "ghcr.io/eclipse-symphony"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
```

Comment:
For GitHub actions, please refer to this PR: https://github.com/eclipse-symphony/symphony/pull/483
some of actions should be `@v4` or `@v5` as the old versions are going to be deprecated

Reply:
fixed